check_repo.py

```diff
@@ -33,6 +32,9 @@ def check_repo():
         if addon_folder[0] != '.':
             addon_path = os.path.join(repo_path, addon_folder)
             error_counter = check_addon.start(error_counter, addon_path, config)
+    
+    if check_addon.comments:
```

Comment:
This means that it will only comment when it find problems? Might be nice to always comment with everything is fine with your commit?
Not sure, @MartijnKaijser for spam analysis ;)

Reply:
This would be the line were you load the config file instead and default to false ;)

Reply:
When all is fine it already shows the green check mark ;)
Option perhaps would be to use labels like "checks passed" for this instead of a comment? If it fails also set "Checks failed" label.
Of course this also needs to unset/set these when situation changes. This allows to filter the open PRs.